### PR TITLE
Add new release volunteers

### PR DESCRIPTION
**Pull request by bboreham**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ Our goal is to provide a new minor release every 4 weeks. This is a new process
 |----------------|--------------------------------------------|---------------------------------------------|
 | v0.1.0         | 2019-07-31                                 | Chris Marchbanks (GitHub: @csmarchbanks)    |
 | v0.2.0         | 2019-08-28                                 | Goutham Veeramachaneni (Github: @gouthamve) |
-| v0.3.0         | 2019-10-09                                 | **searching for volunteer**                 |
+| v0.3.0         | 2019-10-09                                 | Bryan Boreham (@bboreham)                   |
+| v0.4.0         | 2019-11-13                                 | Tom Wilkie (@tomwilkie)                     |
```
Comment (codesome) [inline RELEASE.md]: Currently its counting 5 weeks.
```suggestion
| v0.4.0         | 2019-11-06                                 | Tom Wilkie (@tomwilkie)                     |
```

File: RELEASE.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ Our goal is to provide a new minor release every 4 weeks. This is a new process
...
+| v0.5.0         | 2019-12-11                                 | **searching for volunteer**                 |
```
Comment (codesome) [inline RELEASE.md]: Adjusted for the previous suggestion.
```suggestion
| v0.5.0         | 2019-12-04                                 | **searching for volunteer**                 |
```

File: RELEASE.md (hunk 1 above)
Comment (bboreham) [reply RELEASE.md]: I figured it would be better to do it the week after Promcon EU.

File: RELEASE.md (hunk 2 above)
Comment (bboreham) [reply RELEASE.md]: I believe Tom is at AWS Reinvent that week.

Comment (bboreham) [reply RELEASE.md]: Oops, ignore me - Tom is the month before.

File: RELEASE.md (hunk 1 above)
Comment (codesome) [reply RELEASE.md]: Yup, that makes sense. Didn't think about that!

**Pull request merged**